lib/python/amsterdam-schema/amsterdam_schema/types.py

```diff
@@ -72,7 +72,44 @@ def from_dict(cls, obj: dict):
     @property
     def tables(self) -> typing.List[DatasetTableSchema]:
         """Access the tables within the file"""
-        return [DatasetTableSchema(i, _parent_schema=self) for i in self["tables"]]
```

Comment:
cool. Enige wat ik mij afvraag is of dit iets explicieter kan. Nu laten de parser extra tabellen verzinnen omdat ons dat goed uitkomt. Dit is echter een implementatie keuze.

Handiger zou denk ik zijn dat we een `DatasetTableSchema.nested_tables` property hebben, die dan alle subobjecten teruggeeft. Dan kan een gebruiker van onze library kiezen hoe deze de tabellen wilt uitlezen.

Eventueel dan op hoofdniveau: `DatasetSchema.get_tables(include_nested=True)`?

Reply:
Goede! Ga ik beide toevoegen.

Reply:
:heavy_check_mark: